Gemfile

```diff
@@ -1,10 +1,10 @@
 source "https://rubygems.org"
 
-gem "rails", "~> 5.2.4"
+gem "rails", "~> 6.0.2"
 
 gem "dalli", "~> 2.7"
 gem "govspeak", "~> 6"
-gem "mongoid", "~> 6"
+gem "mongoid", "~> 7.0.5"
```

Comment:
I think in the past upgrading to mongoid 7.x hasn't worked because it [dropped support for Mongo 2.4](https://github.com/mongodb/mongoid/releases/tag/v7.0.0) which we use in production.

Reply:
We can't upgrade to `rails 6` without upgrading to `mongoid 7.x`.
If `mogoid 7.x` doesn't support `Mongo 2.4` that we use in production, we can either:
- upgrade to a Mongo version that is supported by `mongoid 7.x`
- not upgrade to `rails 6`

Do you see other alternatives @thomasleese? 🤔

Reply:
Agreed. But upgrading to Mongo >2.4 seems like a big job. It's probably worth talking to our new Lead Devs, and RE.

Reply:
I think upgrading to Mongo >2.4 is happening as we move over apps to AWS (using DocumentDB), maybe @sengi or @rtrinque can confirm that? If that's the case, perhaps we should prioritise Rails upgrades on apps already migrated over to AWS.

Reply:
I believe we are indeed aiming to move Maslow onto DocumentDB (API-compatible with Mongo 3.6) as part of the AWS move of all the "backend" publisher apps. These aren't moved yet in Production. @camdesgov is leading the project.

Reply:
Should we park this until the migration has taken place? @issyl0 @thomasleese @cbaines

Reply:
I've now changed the Maslow Jenkinsfile to explicitly test against MongoDB 2.4, as previously the tests were sometimes running against MongoDB 3.2, which was why this was working on the CI Jenkins.